new popup eric

nathiwos68 opened the pull request:
adding new popup eric

Commit:
new popup eric

adding new popup eric

cuvaalex reviewed (changes requested):
Do you mean your script work only for one speaker? Please create a generic script that we can use for any speakers or appropriately rename

Otherwise, the pop is fine unless you need to have a cursor change when the mouse is in the title, so people know there are more details, or put an icon near.

Commit:
Speaker Popup

+ I'd moved content of speaker to model/speaker.js to manage easier
+ If speaker description existed the popup will be showed when user
click speaker block
